required/latex-lab/latex-lab-block.dtx

```diff
@@ -720,7 +720,7 @@
 \ProvidesPackage {latex-lab-testphase-block}
                  [\ltlabblockdate\space v\ltlabblockversion\space
                             blockenv implementation]
-%    \end{macrocode}
+%    \end{macrocode} = ignore ,
```

Comment:
Is this supported syntax?

Reply:
I have no idea how that ended up there, thanks for spotting